pkg/web/pages.go

```diff
@@ -84,18 +84,6 @@ func (s *Server) SendEnvelopeForm(c *gin.Context) {
 	c.HTML(http.StatusOK, "send_envelope.html", scene.New(c))
 }
 
-func (s *Server) SendMessageForm(c *gin.Context) {
-	c.HTML(http.StatusOK, "send_message.html", scene.New(c))
-}
-
-func (s *Server) VerifySunriseUser(c *gin.Context) {
-	c.HTML(http.StatusOK, "verify.html", scene.New(c))
-}
-
-func (s *Server) SunriseMessagePreview(c *gin.Context) {
-	c.HTML(http.StatusOK, "view_message.html", scene.New(c))
-}
-
```

Comment:
This will likely have to move to the new Sunrise service, so I've put them all into sunrise.go for easy modification.